helm/resource_release_test.go

```diff
@@ -817,6 +817,27 @@ func TestGetValuesString(t *testing.T) {
 	}
 }
 
+func TestGetListValues(t *testing.T) {
+	d := resourceRelease().Data(nil)
+	err := d.Set("set_list", []interface{}{
+		map[string]interface{}{"name": "foo", "value": []string{"1", "2", "3"}, "type": "string"},
+	})
+	if err != nil {
+		t.Fatalf("error setting values: %s", err)
+		return
+	}
+
+	values, err := getValues(d)
+	if err != nil {
+		t.Fatalf("error getValues: %s", err)
+		return
+	}
+
+	if len(values["foo"].([]interface{})) != 3 {
```

Comment:
Here you are only testing that there is 3 items. I would actually assert that you get back `map[string]interface {}{"foo":[]interface {}{"1", "2", "3"}}`